src/filters/index.js

```diff
@@ -61,7 +61,7 @@ var digitsRE = /(\d{3})(?=\d)/g
 exports.currency = function (value, currency) {
   value = parseFloat(value)
   if (!isFinite(value) || (!value && value !== 0)) return ''
-  currency = currency || '$'
+  currency = currency === false ? '' : currency || '$'
```

Comment:
well, as I said in PR description, `''` is casted to `undefined` :) Is this bug somewhere in parser?  

Otherwise I was going to use empty string as a *no-symbol param* also. Only "ok" choice was to take `false` as *no-symbol param*. Another choice is to remove this slightly discriminating `$` default :smile: , but that would be breaking change.